css/index.css

```diff
@@ -1,3 +1,382 @@
+    display: flex;
+}
+
+.facts-section ul {
+    list-style-type: square;
+    margin: 20px 0 0 20px;
+    line-height: 200%;
+}
+
+.facts-section p {
+    margin-right: 25px;
+}
+
+/* tablet */
+
+@media (max-width:800px) {
+    .container {
+        width: 100%;
+        margin-top: 10px;
+    }
+    /* .computer-logo {
+        display: none;
+      } */
+    .fixed-nav {
+        width: 100%;
+        z-index: 100;
+        background: white;
+    }
+    nav {
+        display: flex;
+        flex-direction: column-reverse;
+        align-content: center;
+    }
+    .great-idea {
+        display: flex;
+        justify-content: center;
+        margin-bottom: 10px;
+    }
+    .tablet-logo-container {
+        display: flex;
+        justify-content: center;
+    }
+    .tablet-logo {
+        display: flex;
+    }
+    .link-container {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 100;
+        background: white;
+        flex-direction: row;
+    }
+    header {
+        justify-content: center;
+        align-items: center;
+        /* height: 100%; */
+        width: 100%;
+        margin-top: 80;
+    }
+    header .headerText {
+        margin-top: 50px;
+    }
+    header img {
+        display: none;
+    }
+    /* .link-container .computer-logo {
+        display: none;
+      } */
+}
+
+.middle-img {
+    width: 100%
+}
+
+/* phone */
+
+@media (max-width:500px) {
+    .container {
+        width: 95%;
+    }
+.container p{
+    margin-bottom: 2rem;
+}
+    nav {
+        height: auto;
+        display: flex;
+        text-align: center;
+        justify-content: center;
+        border-bottom: 1px solid gray;
+        color: #B0B0B0;
+       
+
+        /* justify-content: center; */
+    }
+    nav a {
+        display: flex;
+        height: 100%;
+        width: 100%;
+        text-align: center;
+        border-top: 1px solid gray;
+        justify-content: center;
+        align-items: center;
+        order: 1;
+        color: #B0B0B0;
+        font-size: 1.5rem;
+        margin:0;
+    }
+    .link-container {
+        display: flex;
+        flex-direction: column;
+        height: 435px;
+        align-items: center;
+        text-align: center;
+       
+        
+        
+    }
+    .link-container .logo {
+        justify-content: center;
+        
+    }
+header {
+    height: 21rem;
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 
+}
+    .topSection {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        margin: 0 20px;
+        margin-top: 60px;
+
+        flex-grow: 1;
+        width: 90%;
+        height: auto;
+        padding: 50px 5px;
+    }
+    .middle-img {
+        margin-top: 10;
+    }
+    .bottomSection {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        margin: 0 20px;
```

Comment:
remember, having width set to 100%, and also having margin, will cause it to break out of its container if the container has a max-width. This is resulting in you always having a horizontal scroll bar on your mobile view, rather than it just taking up 100% percent of the screen. What you want here is padding